rules-tests/TypeDeclaration/Rector/Property/PropertyTypeDeclarationRector/Fixture/skip_inherit_doc.php.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\PropertyTypeDeclarationRector\Fixture;
+
+final class A {
+    /** @inheritDoc */
+    public $property = 'test';
+}
```

Comment:
This is not really valid example, as there is no parent class to inherit type from.

Reply:
There's no parent, but should it matter?

```
class A {
    public $prop;
}

class B extends A {
    /** @inheritDoc */
    public $prop;
}
```

Given class `A` is part of vendor code and you're expecting it to have the property type defined, should your class `B` change depending on whether the parent has the property type defined or not? `A` may change at any time (property type might get removed for whatever reason), should it result in a sudden addition of type declaration in `B`?

If so, I'll add inheritance back to the fixture. Not sure what to do with the rector itself in this case though.

Reply:
I've changed the fixture to include a parent class with a typed property, although the question still remains whether a case where parent property is untyped should be supported or not. 

Current implementation doesn't check for parent at all.

Reply:
Thank you!

The previous fixture should be kept and changed.

Reply:
There are now two fixtures, both valid in terms of inheritance. One has the type specified, the other does not.